tests/foreman/cli/test_http_proxy.py

```diff
@@ -0,0 +1,93 @@
+"""Tests for http-proxy hammer command.
+
+:Requirement: HttpProxy
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Repositories
```

Comment:
I think component would be different but it's pending on me and I will update you on it.

Reply:
ok